about.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/abtstyle.css">
+</head>
+<body>
+    <header>
+        <div class="upper_navbar">
+            <ul>
+             <li><a href="#"><img src="images/linkedinlogo.png" alt="Linked in nav item"></a></li>
+             <li><a href="#"><img src="images/twitterlogo.png" alt="Linked in nav item"></a></li>
+             <li><a href="#">English</a></li>
+             <li><a href="#">My Page</a></li>
+             <li><a href="#">Logout</a></li>          
+          </ul>
+        </div>
+        <nav>
+            <button type="button" id="hamburger-btn"><img src="images/hamburger.png" alt="Hamburger button"></button>
+            <div class="desktop-navbar">
+                <a class="logo_link" href="#"><img src="images/Screenshot from 2022-02-16 15-13-24.png" alt="Homifyers logo link"></a>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#contact">Contact</a></li>
+                    <li><a href="#tstmons">Top Customers</a></li>
+                    <li><a href="#brands">Brands</a></li>
+                    <li><a href="#wrhs">Visit</a></li>
+                </ul>
+            </div>
+        </nav>
```

Comment:
- [ ] The top section background image is not aligned properly, kindly consider adding the background to the top section only so it can look like the provided design

|Your design|Provided design|
|-|-|
|![Screenshot_20221028_224454](https://user-images.githubusercontent.com/25227022/198737979-a0956e38-1951-4b34-99a3-c4041db0c8fe.png)|![Screenshot_20221028_224304](https://user-images.githubusercontent.com/25227022/198737831-f2b223ef-aded-4678-b363-96b209dd5715.png)|